docker/openmetadata.yaml

```diff
@@ -125,8 +125,11 @@ database:
   user: ${DB_USER:-openmetadata_user}
   password: ${DB_USER_PASSWORD:-openmetadata_password}
   # the JDBC URL; the database is called openmetadata_db
-  url: jdbc:${DB_SCHEME:-mysql}://${DB_HOST:-localhost}:${DB_PORT:-3306}/${OM_DATABASE:-openmetadata_db}?allowPublicKeyRetrieval=true&useSSL=${DB_USE_SSL:-false}&serverTimezone=UTC
+  url: jdbc:${DB_SCHEME:-mysql}://${DB_HOST:-localhost}:${DB_PORT:-3306}/${OM_DATABASE:-openmetadata_db}?${DB_PARAMS:-allowPublicKeyRetrieval=true&useSSL=false&serverTimezone=UTC}
```

Comment:
@ArtyomyuS Thank for the PR. Any reason to remove DB_USE_SSL and making useSSL=false.
This will prevent others from enabling useSSL option.

Reply:
Hello @harshach , thanks for your comment. The idea was to extract all the params out of the db url specification as MySQL and PostgreSQL have different params. In case of `useSSL` this is supported just by MySQL, in case of PostgreSQL it is: `sslmode`. So I thought that is more appropriate just to specify whatever parameters are required based on the desired db integration.

Reply:
@ArtyomyuS that makes sense. can we remove the useSSL=false from the connection string? otherwise it will override what we are passing through DB_PARAMS right?